CI_YAML.md

```diff
@@ -124,33 +125,155 @@ following are a list of keys that are reserved for special use.
 **Properties** is a Map<String, String> and any special values must be JSON encoded
 (i.e. no trailing commas). Additionally, these strings must be compatible with YAML multiline strings
 
-**$flutter/osx_sdk**: xcode configs including sdk and runtime. **Note**: support on legacy `xcode`/`runtime`
-properties and `xcode` dependency has been deprecated.
+<table>
+  <tr>
+    <td>
+      <b>Property Name</b>
+    </td>
+    <td>
+      <b>Description</b>
+    </td>
+    <td>
+      <b>Default Value</b>
+    </td>
+    <td>
+      <b>Type</b>
+    </td>
+    <td>
+      <b>Example</b>
+    </td>
+  </tr>
+  <tr>
+    <td>add_recipes_cq</td>
+    <td>Whether to add this target to flutter/recipes CQ. This ensures
+changes to flutter/recipes pass on this target before landing.
+    </td>
+    <td>"false"</td>
+    <td>string bool</td>
+    <td>
 
-Example:
 ``` yaml
-$flutter/osx_sdk : >-
-  {
-    "sdk_version": "14e222b",
-    "runtime_versions":
-      [
-        "ios-16-4_14e222b",
-        "ios-16-2_14c18"
-      ]
-  }
+add_recipes_cq: "true"
 ```
+</td>
+  </tr>
+  <tr>
+    <td>cache_name</td>
+    <td>The name identifier of the second layer Engine source cache.
```

Comment:
I am assuming second layer cache is provided by our CI and not LUCI? Or is this something entirely different?

Reply:
Yes. Updated a bit.